packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -3710,6 +3718,8 @@ const ruleset = {
             description: "This rule checks for references that aren't using latest version of common-types.",
             message: "{{error}}",
             severity: "warn",
+            disableForTypeSpec: true,
+            disableForTypeSpecReason: "TODO",
```

Comment:
This one I am not fully sure we have a linter rule for that does exactly that. We have one that ensure you picked a version but not check we use the latest.

Reply:
Also the arm-common-types rule is not currently enabled by default, so I don't think we should disable this yet.

Reply:
No, we should not disable in this PR.